src/decoding/extrinsic_decoder.rs

```diff
@@ -634,7 +659,7 @@ where
             }
 
             Ok::<_, ExtrinsicDecodeError>(ExtrinsicExtensions {
-                transaction_extensions_version,
+                transaction_extensions_version: transaction_extensions_version.unwrap_or(0),
```

Comment:
So this doesn't exist for ExtrinsicType::Bare AFAIU, cool

A bit hard to understand with optio, maybe that could be improved with separate types from `Bare, Signed and General` to avoid this `transaction_extensions_version: Option` but fine for now

Reply:
I pondered separate types but didn't offhand because we don't know the extrinsic version until later so it'd end up with an Option anyways or getting messier converting from one type to another :)